### PR TITLE
[PRM-1926] Added url to Check what you owe page

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -146,6 +146,10 @@ signOut {
   url = "http://localhost:9553/bas-gateway/sign-out-without-state?continue="
 }
 
+whatYouOwe {
+  url = "http://localhost:9152/vat-through-software/what-you-owe"
```
Comment (AndrewRButcher) [inline conf/application.conf:150]: This will need an app-config base PR to override in envs

File: test/views/IndexViewSpec.scala [hunk 2]
```diff
@@ -60,6 +79,20 @@ class IndexViewSpec extends SpecBase with ViewUtils with ViewBehaviours {
       }
     }
 
+    "display button with correct text and link when user owes penalties" when {
+      "user is trader" in {
+        val docWithPenalties = asDocument(applyView(userOwes = true))
+        docWithPenalties.select(Selectors.button).text() shouldBe whatYouOweButtonText
+        docWithPenalties.select(Selectors.button).attr("href").contains("http://localhost:9152/vat-through-software/what-you-owe") shouldBe true
```
Comment (AndrewRButcher) [inline test/views/IndexViewSpec.scala:86]: Avoiding using shouldBe true
```suggestion
        docWithPenalties.select(Selectors.button).attr("href") shouldBe "http://localhost:9152/vat-through-software/what-you-owe"
```

File: test/views/IndexViewSpec.scala [hunk 3]
```diff
@@ -60,6 +79,20 @@ class IndexViewSpec extends SpecBase with ViewUtils with ViewBehaviours {
...
+      }
+
+      "user is agent" in {
+        val docWithPenalties = asDocument(applyView(userOwes = true, isUserAgent = true))
+        docWithPenalties.select(Selectors.button).text() shouldBe whatYouOweButtonAgentText
+        docWithPenalties.select(Selectors.button).attr("href").contains("http://localhost:9152/vat-through-software/what-you-owe") shouldBe true
```
Comment (AndrewRButcher) [inline test/views/IndexViewSpec.scala:92]: Avoiding using shouldBe true
```suggestion
        docWithPenalties.select(Selectors.button).attr("href") shouldBe "http://localhost:9152/vat-through-software/what-you-owe"
```

**Review by AndrewRButcher — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 1 above)
Comment (AndrewRButcher) [reply conf/application.conf]: app config base pr created and config moved to urls section
https://github.com/hmrc/app-config-base/pull/7771

**Comment by AndrewRButcher** [comment]
Rebased ✅

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**